### PR TITLE
Add benchmark using The Cancer Genome Atlas transcriptomic data

### PR DESCRIPTION
**Pull request by fvalle1**
!! I have not any GPU hardware to test this, please verify that this works properly on GPUs before merging.

This PR adds a benchmark using to OpenForBC with biological trascriptome data from [TCGA](https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga). It uses tensorflow to interact with GPUs in the same way as your MNIST benchmark. Presets on CPU have been tested properly.

This PR locally passed
- [x] poetry run pytest
- [x] poetry run flake8

### CODE REVIEW

**Comment by leggerf** [comment]
hi Filippo, many thanks for this PR. I gave a quick try on our T4, and benchmark fails with this error:

breast/topsbm/topsbm_level_1_word-dist.csv
Running "TCGA_topicmodeling" preset "inference"
(venv) $ ./TCGA_topicmodeling_benchmark.py gpu inference -l 100
2022-03-29 08:41:24.777518: I tensorflow/core/platform/cpu_feature_guard.cc:151] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2022-03-29 08:41:26.960189: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1525] Created device /job:localhost/replica:0/task:0/device:GPU:0 with 13797 MB memory:  -> device: 0, name: Tesla T4, pci bus id: 0000:41:00.0, compute capability: 7.5
nvidia-smi has not been found. GPU support is excluded.
Traceback (most recent call last):
  File "/home/jovyan/OpenForBC-Benchmark/benchmarks/TCGA_topicmodeling/./TCGA_topicmodeling_benchmark.py", line 412, in <module>
    nvidia_smi.nvmlInit()
NameError: name 'nvidia_smi' is not defined
Benchmark "TCGA_topicmodeling" preset "inference" command "./TCGA_topicmodeling_benchmark.py gpu inference -l 100" failed
Task (venv) $ ./TCGA_topicmodeling_benchmark.py gpu inference -l 100 failed with return code 1
ERROR: Benchmark "TCGA_topicmodeling" failed

nvidia-smi is on the system, so maybe some setup is not correct in the venv?

**Comment by fvalle1** [comment]
Hi @leggerf,

You are right, I was missing *nvidia-smi* import

File: benchmarks/TCGA_topicmodeling/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+breast/
+*.tar
+*.gz
+stats*.txt
```
Comment (gabrielefronze) [inline benchmarks/TCGA_topicmodeling/.gitignore:4]: Can you please check newlines at end of files in all files? Thank you!

**Review by gabrielefronze — CHANGES_REQUESTED** [review]
Thanks! Just check the presence of a newline at the end of files and we should be good to merge!

**Comment by gabrielefronze** [comment]
Thanks @fvalle1 ! Looks good to me!

**Pull request merged**